app/views/state_file/questions/id_review/edit.html.erb

```diff
@@ -24,5 +24,15 @@
   </section>
 
 
+  <% if current_intake.has_filing_requirement? && !current_intake.has_blind_filer? %>
+    <div id="permanent-building-fund" class="white-group">
+      <div class="spacing-below-5">
+        <p class="text--bold spacing-below-5"><%=t(".permanent_building_fund_tax") %></p>
+        <p><%= current_intake.received_id_public_assistance == "no" ? t("general.affirmative") : t("general.negative") %></p>
```

Comment:
can we use the `current_intake.received_id_public_assistance_no?` here instead of the comparison